### DISCUSSION TITLE
Release 1.4 Date

### DISCUSSION THREAD

**Discussion by Aubi0ne** [google/model-viewer#1841] [Q&A] [answered]
Hello,

Do you know when the next release 1.4 will be available ? I'm waiting for the ar wall placement feature ^^

**Comment by elalish** [comment, marked as answer]
Looks like it's your lucky day: https://github.com/google/model-viewer/releases/tag/v1.4.0

**Reply by Aubi0ne** [reply]
Wow, what a sense of timing ! And it now works on ios chrome. Champagne ! 🍾🥳🎉
